NGitLab.Mock/Clients/GitLabClient.cs

```diff
@@ -37,6 +39,12 @@ public GitLabClient(ClientContext context)
 
         public IMergeRequestClient MergeRequests => new MergeRequestClient(Context);
 
+        public IGraphQLClient GraphQL
+        {
+            get => _overrideGraphQLClient ?? new GraphQLClient(Context);
```

Comment:
Couldn't we store the new instance if `_overrideGraphQLClient` was null?
`get => _overrideGraphQLClient ??= new GraphQLClient(Context);`